hazelcast-jet-core/src/main/java/com/hazelcast/jet/windowing/WindowOperations.java

```diff
@@ -62,7 +62,7 @@ private WindowOperations() {
     /**
      * Returns an operation that counts the items in the window.
      */
-    public static <T> WindowOperation<T, ?, Long> summingToLong(@Nonnull Distributed.ToLongFunction<T> mapper) {
+    public static <T> WindowOperation<T, ?, Long> summingToLong(@Nonnull Distributed.ToLongFunction<? super T> mapper) {
```

Comment:
No. I just did it, because I always do it with mappers. Yes, in this case, T isn't used anywhere else.